src/rating-indicator.scss

```diff
@@ -0,0 +1,119 @@
+@import "./new-settings";
+@import "./mixins";
+@import "./icon";
+
+$block: #{$fd-namespace}-rating-indicator;
+
+.#{$block} {
+  $ratedColor: var(--sapContent_RatedColor);
+  $unratedColor: var(--sapContent_UnratedColor);
+  $active-star-hover-opacity-transaction: opacity $fd-animation-speed ease-in;
+  $fd-rating-indicator-sizes: (
+    "cozy": ("fontSize": 1.5rem, "spacingBetweenIcons": 0.1875rem, "controlMargin": 0.8125rem 0),
+    "compact": ("fontSize": 1rem, "spacingBetweenIcons": 0.125rem, "controlMargin": 0.5rem 0),
+    "condensed": ("fontSize": 1rem, "spacingBetweenIcons": 0.125rem),
+    "xs": ("fontSize": 0.75rem, "spacingBetweenIcons": 1px),
+    "s": ("fontSize": 1rem, "spacingBetweenIcons": 0.125rem),
+    "m": ("fontSize": 1.35rem, "spacingBetweenIcons": 0.1875rem),
+    "l": ("fontSize": 2rem, "spacingBetweenIcons": 0.25rem)
+  );
+  $defaultSize: map-get($fd-rating-indicator-sizes, 'm');
+  $disabledOpacity: var(--sapContent_DisabledOpacity);
```

Comment:
The idea of creating a variable is that it will be used in multiple places in the code and since this one is used only once you don't need to put the value in a variable

Reply:
fixed